lib/php/lib/Server/TSSLServerSocket.php

```diff
@@ -49,6 +49,10 @@ public function __construct($host = 'localhost', $port = 9090, $context = null)
     {
         $ssl_host = $this->getSSLHost($host);
         parent::__construct($ssl_host, $port);
+        // Initialize a stream context if not provided
+        if ($context === null) {
+            $context = stream_context_create();
```

Comment:
stream_socket_server expects to receive resource instead of null. Perhaps thats why warning is shutted down by @